osc/commandline.py

```diff
@@ -5015,10 +5015,16 @@ def _commit(self, subcmd, opts, args):
         skip_local_service_run = False
         if not conf.config['local_service_run'] or opts.skip_local_service_run:
             skip_local_service_run = True
-        arg_list = args[:]
-        for arg in arg_list:
+
+        for arg in args.copy():
             if conf.config['do_package_tracking'] and is_project_dir(arg):
                 prj = Project(arg)
+
+                if prj.scm_url:
+                    print("WARNING: Skipping project '{}' because it is managed in scm (git)".format(prj.name))
```

Comment:
please output also the scm_url, so that the user has a chance to follow up

Reply:
Right, display some context. Done.